package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-lifeline",
-  "version": "3.0.9",
+  "version": "3.0.10",
```

Comment:
I bump this when publishing, please reset in the PR (other wise I have conflicts)